emperor/support_files/js/emperor.js

```diff
@@ -886,20 +887,23 @@ function toggleTaxaLabels(){
 	}
 }
 
-/* Turn on and off the spheres representing the biplots on screen */
-function toggleBiplotVisibility(){
+/* Turn on and off the spheres representing the biplots on screen
+
+   If the toggleArrow variable is true, then only the visibility of the taxa vectors will be altered
+ */
+function toggleBiplotVisibility(toggleArrow){
 	// reduce the opacity to zero if the element should be off or to 0.5
 	// if the element is supposed to be present; 0.5 is the default value
-	if(!document.biplotsvisibility.elements[0].checked){
-		for (index in g_plotTaxa){
-			g_mainScene.remove(g_plotTaxa[index]);
-		}
-	}
-	else{
-		for (index in g_plotTaxa){
-			g_mainScene.add(g_plotTaxa[index])
-		}
-	}
+    toggleArrow = typeof toggleArrow !== 'undefined' ? toggleArrow : false;
```

Comment:
huh, well, I guess it didn't reduce the LOC, but at least the forloop is much simpler :)